libs/spirit-islander/data-access/src/lib/app-state.service.ts

```diff
@@ -32,30 +32,47 @@ interface AppState {
 export class AppStateService {
   private readonly _oldKey = 'CONFIG_NEW';
   private readonly _key = 'CONFIG';
-  private _config: Config = this._getConfig();
+  private _config: Config = this._getConfig(); // TODO: consider initializing _state in the constructor so this _config doesn't have to be a long-lived class member
```

Comment:
Address this comment

Reply:
Mentioned [here](https://github.com/johnnycopes/atocha/issues/280#issuecomment-1490275264)